src/pages/UserHome.jsx

```diff
@@ -117,7 +117,7 @@ export default function UserHome() {
           component="h2"
           gutterBottom
           className={classes.noScholarshipsAdded}>
-          No Scholarships Added Yet
+          <div>{scholarships.length === 0 && 'No Scholarships added.'}</div>
```

Comment:
This isn't the only piece of the code that needs to be conditionally rendered. Look at Figma design and compare what the page looks like [with scholarships](https://www.figma.com/file/nwrW3UebcvSWyQZGb5ixNr/DreamScholar?node-id=562%3A1691) and [without scholarships](https://www.figma.com/file/nwrW3UebcvSWyQZGb5ixNr/DreamScholar?node-id=562%3A1809)